src/NuGet.Core/NuGet.Configuration/Settings/Settings.cs

```diff
@@ -152,7 +152,7 @@ public string ConfigFilePath
                 {
                     validSettingFiles.AddRange(
                         GetSettingsFileNames(root)
-                            .Select(f => ReadSettings(root, f))
+                            .Select(f => ReadSettings(root, f, isMachineWideSettings: false, throwOnFailure: false))
```

Comment:
Coding guideline. We do not prefer default values for function parameters. And, this is a small change where we can avoid using default values